ui-v2/app/templates/dc/intentions/index.hbs

```diff
@@ -33,15 +37,16 @@
             data-test-intention-filter="true"
             @value={{search}}
             @onsearch={{action (mut search) value="target.value"}}
-            @selected={{filter.selected}}
-            @options={{filter.items}}
-            @onchange={{action (mut filterBy) value='target.value'}}
+            @secondary="sort"
+            @selected={{sort.selected}}
+            @options={{sort.items}}
+            @onchange={{action (mut sortBy) value='target.value'}}
```

Comment:
@kaxcode I noticed that this should be `target.selected.key`:

https://github.com/hashicorp/consul/blob/77b4d8f42a467cef78248cc1ef36f4c33306af3a/ui-v2/app/templates/dc/services/index.hbs#L30

Not entirely sure why the tests on this PR didn't catch this, as the tests caught it after rebasing this onto the branch that I'm on.

I'd probably double check the other PRs you have open for ordering also!

Reply:
Thanks for catching that! I had some issues reusing this branch. I've made a new branch and here's the PR #8398.